features/networking/egress-ip.feature

```diff
@@ -130,6 +130,8 @@ Feature: Egress IP related features
     """
     as admin I successfully merge patch resource "hostsubnet/<%= node.name %>" with:
       | {"egressCIDRs":null}   |
+    as admin I successfully merge patch resource "hostsubnet/<%= node.name %>" with:
+      | {"egressIPs":null}   |
```

Comment:
Is it possible to do the two patches in one command ?

Reply:
Thanks for reviewing, no, it cannot be done in one command.